rust/arrow/src/datatypes.rs

```diff
@@ -1289,6 +1417,46 @@ impl Schema {
         Self { fields, metadata }
     }
 
+    pub fn try_merge(schemas: &Vec<Self>) -> Result<Self> {
```

Comment:
May you please add a doc for this

Reply:
ha, good catch, i added a doc for this, but it was mistakenly added to Field's try_merge method. I have fixed and also added doc for Field's try_merge method as well.

Reply:
Thanks, I'll await a review from another commiter, then we can merge